src/stan_math_backend/Expression_gen.ml

```diff
@@ -203,8 +203,19 @@ and gen_distribution_app f =
 (* assumes everything well formed from parser checks *)
```

Comment:
Question: how is the code gen currently organised? I see that the expression generation is in its own file. Does the other file only contain code gen for statements? In that case, would it make sense to call it Statement_gen?
Or better still, can we call them Expression_code_gen and Statement_code_gen?

Reply:
No, it contains all of the rest of the code gen. I'd pull out the statement gen into another file if you think that'd help. The expression_gen was kept separate as a sort of learning exercise for Bob, and I'm not unhappy with that split out, so it might make sense to go ahead and do the same for statements?

Reply:
Sure! Sounds good! That sort of organization might make it a bit easier to browse this beast.